end_to_end_tests/generated_client.py

```diff
@@ -45,6 +45,16 @@ def import_module(self, module_path: str) -> Any:
         """Attempt to import a module from the generated code."""
         return importlib.import_module(f"{self.base_module}{module_path}")
 
+    def import_symbol(self, module_path: str, name: str) -> Any:
+        module = self.import_module(module_path)
+        try:
+            return getattr(module, name)
+        except AttributeError:
+            existing = ", ".join(name for name in dir(module) if not name.startswith("_"))
+            assert False, (
+                f"Couldn't find import \"{name}\" in \"{self.base_module}{module_path}\"."
+                f" Available imports in that module are: {existing}"
+            )
```

Comment:
Just some nicer failure output. In a case like the one I mentioned above, where the generator has created an incorrect type name, if you try to import `NameOfThing` you'll see something like:

`AssertionError: Couldn't find import "NameOfThing" in "testapi_client.models". Available imports in that module are: NameOfThingType0, SomeOtherType, [etc.]`